distrobuilder/main.go

```diff
@@ -587,32 +566,80 @@ EOF
 
 	mkdir -p /run/systemd/system/default.target.wants
 	ln -s /run/systemd/system/network-device-down.service /run/systemd/system/default.target.wants/network-device-down.service
-)
+}
 
-# Fix for systemd-networkd
-(
-	. /etc/os-release
+# fix_systemd_override_global generates the global systemd override
+fix_systemd_override_global() {
+	# Setup global override
+	dropin_dir="/run/systemd/system/service.d"
+	mkdir -p "${dropin_dir}"
+	echo "[Service]" > "${dropin_dir}/lxc.conf"
+	[ "${systemd_version}" -ge 247 ] && echo "ProtectProc=default" >> "${dropin_dir}/lxc.conf"
+	echo "ProtectControlGroups=no" >> "${dropin_dir}/lxc.conf"
+	echo "ProtectKernelTunables=no" >> "${dropin_dir}/lxc.conf"
+}
```

Comment:
Not sure if we *really* need this. We could just call `fix_systemd_override_unit system/service.d` instead.

Reply:
Done